src/libstd/sys/windows/rand.rs

```diff
@@ -1,7 +1,10 @@
 use crate::io;
 use crate::mem;
+#[cfg(target_os = "uwp")]
```

Comment:
For something like this I tend to recommend to just put the import intot he `#[cfg]`'d function below if possible to avoid the `#[cfg]` annotation at the top here

Reply:
Will do